app/views/common_parameters/index.html.erb

```diff
@@ -3,22 +3,23 @@
 <% title_actions display_link_if_authorized(_("New Parameter"), hash_for_new_common_parameter_path),
                  documentation_button('4.2.3Parameters') %>
 
-<table class="table table-bordered table-striped table-two-pane table-fixed">
+
+<table class="table table-bordered table-striped table-two-pane table-hover">
```

Comment:
I think when I did it trunc_with_tooltip was the only option because the table wouldn't do it, so feel free to revert it